packages/fx-core/src/plugins/resource/bot/v2/mapping.ts

```diff
@@ -55,6 +56,15 @@ const projectFileMap = new Map<Runtime, (appName: string) => string>([
   [Runtime.Dotnet, (appName: string) => `${appName}.csproj`],
 ]);
 
+export const moduleMap: { [key: string]: string } = {
+  [ServiceType.Functions]: BicepModules.Functions,
+};
+
+export const moduleMapCapitalized = Object.keys(moduleMap).reduce(
```

Comment:
Nit: From my perspective, the `reduce` used here is not so straightforward and kind of not performant. I may choose to just use a simple loop.

Reply:
Removed this map in latest changes. Thanks for the suggestion.